#assertion
print(1)
assert 2>4
print(2)
assert 1==3
print(3)





try:
	num=5/0
except:
	print(" there is a error")
	#raise






name="123"
#raise NameError("invalid name") 